pkg/nsx/services/vpc/vpc.go

```diff
@@ -611,6 +652,32 @@ func (s *VPCService) CreateOrUpdateVPC(obj *v1alpha1.NetworkInfo) (*model.Vpc, *
 	}
 
 	s.VpcStore.Add(&newVpc)
+
+	// Check LBS realization
+	if createdLBS != nil {
+		newLBS, err := s.NSXClient.VPCLBSClient.Get(nc.Org, nc.NsxtProject, *createdVpc.Id, *createdLBS.Id)
+		if err != nil {
+			log.Error(err, "failed to read LBS object after creating or updating", "LBS", createdLBS.Id)
+			return nil, nil, err
+		}
+		s.LbsStore.Add(&newLBS)
+
+		log.V(2).Info("check LBS realization state", "LBS", *createdLBS.Id)
+		realizeService := realizestate.InitializeRealizeState(s.Service)
+		if err = realizeService.CheckRealizeState(util.NSXTLBVSDefaultRetry, *newLBS.Path, ""); err != nil {
+			log.Error(err, "failed to check LBS realization state", "LBS", *createdLBS.Id)
+			if realizestate.IsRealizeStateError(err) {
+				log.Error(err, "the created LBS is in error realization state, cleaning the resource", "LBS", *createdLBS.Id)
+				// delete the nsx vpc object and re-created in next loop
+				if err := s.DeleteVPC(*newVpc.Path); err != nil {
```

Comment:
So if only lb not realized, must delete the whole vpc, it a must? Can we optimize it by not deleting it, rather only try lbs realization again?

Reply:
lbs realization timeout is longer than vpc, thus it is enough.
lbs is always created with vpc, thus we should use deleteVPC to remove them.

Reply:
Just ask danyang, for lbs realization error, it may due to 'transaction concurrent update' which will be recovered after 5 minutes. so she suggest to wait for 5 minutes for lbs realization error. the NSXTLBVSDefaultRetry wait period is 30seconds?

Reply:
@TaoZou1 30s is from NCP. Do you think we should change it to 5min in nsx-operator?